account_payment_order/models/account_payment_order.py

```diff
@@ -344,6 +348,13 @@ def draft2open(self):
                         "paylines": payline,
                         "total": payline.amount_currency,
                     }
+            # Raise errors that happened on the validation process
+            if payline_err_text:
+                raise UserError(
+                    _("A bunch of validation didn't passed\n")
```

Comment:
```suggestion
                    _("There's at least one validation error:\n")
```

Reply:
Is there a way to check pre-commit stuff before actually the PR check ?

Reply:
You can install pre-commit locally. Having pipx installed (`pip install pipx`), do `pipx install pre-commit`. Then, being on your repo directory, do `pre-commit install -f` for installing a hook. Now, each `git commit` on that repo, will launch first the pre-commit stuff.